pkg/apis/org/v1/che_types.go

```diff
@@ -593,7 +593,9 @@ type CheClusterSpecImagePuller struct {
 	// When set to `false`, the KubernetesImagePuller object will be deleted, and the Operator will be uninstalled,
 	// regardless of whether a spec is provided.
 	//
-	// Note that while this the Operator and its behavior is community-supported, its payload may be commercially-supported
+	// If the `spec.images` field is empty, a set of recommended workspace-related images will be automatically detected and
```

Comment:
Pls, remove empty line above. It breaks formatting in yaml files

Reply:
yes, fixed